src/client/Microsoft.Identity.Client/AuthScheme/TokenType.cs

```diff
@@ -32,6 +32,12 @@ internal enum TokenType
         /// Extension token type.
         /// Extension = 5
         /// </summary>
-        Extension = 5
+        Extension = 5,
+
+        /// <summary>
+        /// MTLS token type.
```

Comment:
Let's pls use the term `mTLS PoP` everywhere, to avoid confusion with mTLS certs.